src/components/checkbox/checkbox.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import Checkbox from './checkbox';
+
+export default {
+  component: Checkbox,
+  title: 'Components/Checkbox',
+  parameters: {
+    layout: 'fullscreen',
+  },
+} as ComponentMeta<typeof Checkbox>;
+
+const Template: ComponentStory<typeof Checkbox> = (args) => (
+  <Checkbox {...args} />
+);
+export const Regular = Template.bind({});
+Regular.args = {
+  label: 'Satellytes',
+};
+Regular.parameters = {
+  backgrounds: {
+    values: [
+      { name: 'black', value: '#000' },
+      { name: 'white (figma)', value: '#F2F2F2;' },
+    ],
+  },
+};
```

Comment:
Thanks for figuring this out. Is it specific to this component, though? Do we maybe want these to be our global storybook background options? I would say so, yes.
Feel free to add them to the exported `parameters` in `.storybook/preview.js` to make them global :raised_hands:

Reply:
We agree that we are focusing on the light theme for now so there is no need to have these settings just yet. We will add them when we tackle the dark theme.